src/renderers/shaders/ShaderLib/depth_vert.glsl

```diff
@@ -12,10 +12,18 @@ void main() {
 
 	#include <skinbase_vertex>
 
+	#ifdef USE_DISPLACEMENTMAP
```

Comment:
If you want to, you can squash the commits into one clean one. It is not required, though.

Nice job. Thanks.